transitions/sequence.h

```diff
@@ -0,0 +1,45 @@
+#ifndef TRANSITIONS_SEQUENCE_H
+#define TRANSITIONS_SEQUENCE_H
+
+#include "random.h"
+
+// Usage: TrSequence<TR1, TR2, ...>
+// TR1, TR2: TRANSITION
+// return value: TRANSITION
+// transition options used in sequence
+
+template<class... TRANSITION>
+class TrSequence {
+public:
+  void begin() {
+    begin_ = true;
+    n_ = (n_ + 1) % sizeof...(TRANSITION);
```

Comment:
This increment should happen in run() if begin_ is true.

Reply:
moved